### PR TITLE
Updates to address comments from WG

### PR DESCRIPTION
**Pull request by tireddy2**

### CODE REVIEW

File: draft-ietf-dnsop-structured-dns-error-page.xml [hunk 1]
```diff
@@ -0,0 +1,1003 @@
...
+    <workgroup>DNSOP WG</workgroup>
+
+    <abstract>
+      <t>DNS filtering is widely deployed for network security, but filtered
+      DNS responses lack information for the end user to understand the reason
+      for the filtering. Existing mechanisms to provide detail to end users
+      cause harm especially if the blocked DNS response is to an HTTPS
+      website.</t>
+
+      <t>This document updates the EXTRA-TEXT field of Extended DNS Error to
```
Comment (boucadair) [inline draft-ietf-dnsop-structured-dns-error-page.xml:120]: I suggest to merge these two sentence to highlight what we actually update in 8914:

This document updates RFC 8914 by structuring the EXTRA-TEXT field of the Extended DNS Error to provide details on the DNS filtering. Such details can be parsed by the client and displayed, logged, or used for other purposes. Other than that, this document does not change any thing written in RFC 8914.

"This document updates RFC 8914 by structuring the EXTRA-TEXT field of the Extended DNS Error to provide details on the DNS filtering. Such details can be parsed by the client and displayed, logged, or used for other purposes. Other than that, this document does not change any thing written in RFC 8914."

File: draft-ietf-dnsop-structured-dns-error-page.xml [hunk 2]
```diff
@@ -0,0 +1,1003 @@
...
+      provide such detail.</t>
+
+      <t>One of the other benefits of this approach is to eliminate the need
+      to "spoof" block pages for HTTPS resources. This is achieved since
+      clients implementing this approach would be able to display a meaningful
+      error message, and would not need to connect to such a block page. This
+      approach thus avoids the need to install a local root certificate
+      authority on those IT-managed devices.</t>
+
+      <t>This document describes a format for computer-parsable data in the
```
Comment (boucadair) [inline draft-ietf-dnsop-structured-dns-error-page.xml:226]: We need to add a warning here that this part is updating 8914, which says: 

"This information is intended for human consumption (not automated parsing)."

**Review by boucadair — CHANGES_REQUESTED** [review]

**Comment by boucadair** [comment]
Thanks Tiru. 

One very very minor edit, we can update the example to be displayed as follows:

```
{
  "c": [
    "tel:+358-555-1234567",
    "sips:bob@bobphone.example.com",
    "https://ticket.example.com?d=example.org&t=1650560748"
  ],
  "j": "malware present for 23 days",
  "s": 1,
  "o": "example.net Filtering Service"
}
```

**Comment by tireddy2** [comment]
Thanks Med. Fixed all the issues.

**Pull request merged**